Production_scripts/Process_extracts/Produce_extract_tests/extract_homelessness_tests.R

```diff
@@ -0,0 +1,43 @@
+####################################################
+# Name of file - extract_homelessness_tests.R
+# Original Authors - Bateman McBride
+# Original Date - July 2022
+# Written/run on - RStudio Server
+# Version of R - 3.6.1
+# Description - Produce tests for source linkage files:
+#               homelessness processed file.
+#####################################################
+
+# Load packages
+library(createslf)
+library(openxlsx)
+library(slfhelper)
+
+# Read in Data-----------------------------------------
+
+year <- "1920"
+
+# Read new data file
+new_data <- readr::read_rds(get_source_extract_path(year, "Homelessness", ext = "rds")) %>%
+  mutate(smrtype = stringr::str_trim(smrtype, "both"))
```

Comment:
```suggestion
new_data <- readr::read_rds(get_source_extract_path(year, "Homelessness", ext = "rds"))
```
Not needed for new version of tests to match SPSS output.

Reply:
Changed this line